index.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express()
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  next();
+});
+
+const Game = require('./src/game.js')
+
+app.get('/game',(req, res) => {
+  Game.join(req.query.token)
+  .then(game => {
+      res.send(game)
+    })
+  .catch(error => console.error(error))
+})
+
+app.get('/game/:gameId/:playerId', (req, res) => {
```

Comment:
1. this endpoint was not part of the requirements, for this task you should only implement the "create" and "join" functionalities.
2. endpoint definition should be: '/game/:gameId/player/:playerId'
3. this could be enhanced with destructuring:
```javascript
  const { gameId, playerId }  = req.params;
```
4. do not push console.logs
5. status code 500 is used for server errors, for not found use 404

Reply:
ok.
And this endpoint, I made it because I wanted to show some result when  endpoint join was active.